packages/core/src/input/input-file-resolver.ts

```diff
@@ -119,7 +119,7 @@ export class InputFileResolver {
   }
 
   private async expandPattern(globbingExpression: string, logAboutUselessPatterns: boolean): Promise<string[]> {
-    if (RegExp('(:\\d+){4}$').exec(globbingExpression)) {
+    if (RegExp(/(:\d+){4}/).exec(globbingExpression)) {
```

Comment:
`/(:\d+){4}/` not `RegExp(/(:\d+){4}/)` ;)

Reply:
Using `globExpression.match(/(:\d+){4}/)` causes an `eslint` error telling me to use `RegExp` instead because `RegExp` is apparently faster?

Reply:
isn't it saying to use /regex/.exec(words)  instead of (words).match(regex)?

Reply:
Done :)